app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -808,7 +810,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         newTab: Boolean,
         engine: SearchEngine?,
         forceSearch: Boolean,
-        flags: EngineSession.LoadUrlFlags = EngineSession.LoadUrlFlags.none()
+        flags: EngineSession.LoadUrlFlags = EngineSession.LoadUrlFlags.none(),
+        requestDesktopMode: Boolean? = null
```

Comment:
I think this should default to `false` too

Reply:
Thanks!